utils/update-next.js

```diff
@@ -18,22 +32,101 @@ function exec(cmd, args, opts) {
   return output;
 }
 
-module.exports = async ({ github, context } = {}) => {
-  process.env.COREPACK_ENABLE_STRICT = '0';
-  const newVersion = exec('npm', ['view', 'next', 'dist-tags.latest']);
-  const branch = `next-${newVersion.replaceAll('.', '-')}`;
+function updatePackageJson(file, newVersion) {
+  const pkgJson = JSON.parse(readFileSync(file, 'utf-8'));
 
-  if (
-    github &&
-    exec('git', ['ls-remote', '--heads', 'origin', branch]).toString().trim()
-  ) {
-    console.log(`Branch ${branch} already exists, skipping update.`);
-    return;
+  if (pkgJson.ignoreNextjsUpdates) {
```

Comment:
This `ignoreNextjsUpdates` value was removed from over 100 fixtures in this PR so that the `canary` fixtures can be pinned. Any fixture with this value that was already pinned to an old version was left in place.